boards/lobaro-lorabox/include/periph_conf.h

```diff
@@ -0,0 +1,237 @@
+/*
+ * Copyright (C) 2014-2016 Freie Universität Berlin
+ *               2018 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_lorabox Lobaro lorabox
+ * @ingroup     boards_lorabox
+ * @brief       Support for the Lobaro lorabox with stm32l151cb
+ * @{
+ *
+ * @file
+ * @brief       Peripheral MCU configuration for theLobaro lorabox board
+ *
+ * @author      Thomas Eichinger <thomas.eichinger@fu-berlin.de>
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author      Kevin Weiss <kevin.weiss@haw-hamburg.de>
+ * @author      Leandro Lanzieri <leandro.lanzieri@haw-hamburg.de>
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include "periph_cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+/**
+ * @name    xtimer configuration
+ * @{
+ */
+#define XTIMER_WIDTH        (16)
+#define XTIMER_BACKOFF      (50)
+#define XTIMER_ISR_BACKOFF  (40)
+/** @} */
+
+/**
+ * @name Clock system configuration
+ * @{
+ **/
+#define CLOCK_HSI           (16000000U)             /* frequency of internal oscillator */
+#define CLOCK_CORECLOCK     (32000000U)             /* targeted core clock frequency */
+/*
+ * 0: no external low speed crystal available,
+ * 1: external crystal available (always 32.768kHz)
+ *
+ * LSE might not be available by default in early (C-01) Nucleo boards.
+ * For newer revisions, an LSE crystal is present and CLOCK_LSE can be set to 1
+ * if one wants to use it.
+ */
+#ifndef CLOCK_LSE
+#define CLOCK_LSE           (1)
```

Comment:
Have you tested this? You should also remove/modify this comment since it comes from a Nucleo board.